cfgmgr/intfmgr.cpp

```diff
@@ -120,16 +120,18 @@ void IntfMgr::setIntfIp(const string &alias, const string &opCmd,
             if (!enableIpv6Flag(alias))
             {
                 SWSS_LOG_ERROR("Failed to enable IPv6 on interface %s", alias.c_str());
-                return;
+                return false;
             }
             ret = swss::exec(cmd.str(), res);
         }
 
         if (ret)
         {
             SWSS_LOG_ERROR("Command '%s' failed with rc %d", cmd.str().c_str(), ret);
+            return false;
```

Comment:
@saiarcot895 , this cannot be indefinitely retried. there can be cases where ip address is assigned and interface is deleted, like sub-interface etc. I suggest to remove the retry logic and fix the current issue with just teamsync changes

Reply:
Reverted the commit that added that logic, and the only changes present are for teamsyncd.